### ISSUE TITLE
/details/contribute/ submit not working

### ISSUE THREAD

**Issue by jackharrhy** [toobigtoignore/issf#77]
#76

To reproduce:

Head to as a logged in user:
http://localhost/details/contribute/

Fill in at least first name and last name, will redirect to:

http://localhost/details/who/basic/

Responds with:

```
ValueError at /details/who/basic/
The view details.views.who_basic didn't return an HttpResponse object. It returned None instead.
```

**Comment by nint8835** [comment]
Issue occurs due to view expecting request to be submitted via AJAX, but it isn't. Issue is fixable by removing the check for is ajax, however this produces a result that appears to be undesired (user lands on page containing json details about the response). Issue appears to be in the javascript/html submitting the request not submitting it via AJAX.

**Comment by nint8835** [comment]
Issue was indeed in the javascript/html, being solved by Jack.

**Comment by nint8835** [comment]
Resolved by PR #82
